scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1727,3 +1735,159 @@ def test_len_and_bool():
     assert rotation_multi_one
     assert rotation_multi
     assert rotation_single
+
+
+def test_from_davenport_single_rotation():
+    axis = [0, 0, 1]
+    quat = Rotation.from_davenport(axis, 'extrinsic', 90, 
+                                   degrees=True).as_quat()
+    expected_quat = np.array([0, 0, 1, 1]) / np.sqrt(2)
+    assert_allclose(quat, expected_quat)
+
+
+def test_from_davenport_one_or_two_axes():
+    ez = [0, 0, 1]
+    ey = [0, 1, 0]
+    rotz = Rotation.from_davenport(ez, 'e', 5)
+    rotz2 = Rotation.from_davenport([ez], 'e', [5])
+
+    assert_allclose(rotz.as_quat(canonical=True),
```

Comment:
I would suggest comparing with `from_rotvec`, I think it will be more convincing that way.

Reply:
Done!